src/ResourceManager/Network/Commands.Network/Models/PSIpFlowVerify.cs

```diff
@@ -20,9 +20,9 @@ public class PSIpFlowVerify
         public string Direction { get; set; }
         public string LocalIPAddress { get; set; }
         public string LocalPort { get; set; }
-        public uint Protocol { get; set; }
-        public uint RemoteIPAddress { get; set; }
-        public uint RemotePort { get; set; }
+        public int Protocol { get; set; }
```

Comment:
this file is not even being used... should be removed. will check with Irina and follow up